### DISCUSSION TITLE
week-select plugin placeholder

### DISCUSSION THREAD

**Discussion by bodek98** [ankurk91/vue-flatpickr-component#281] [Q&A]
Hi, i have problem with placeholder in flatpickr component. I used  weekSelectPlugin like below:
const config = {
  defaultDate: store.getSelectedDate(),
  position: "auto",
  plugins: [new weekSelectPlugin({})],
  locale: {
    ...Polish,
  },
  }

![image](https://github.com/ankurk91/vue-flatpickr-component/assets/94081512/f237e87d-2323-45bb-baea-c477b1617c05)

Is there any way to display placeholder in pattern like below?
firstDayOfWeek (03.07.2023) - lastDayOfWeek (09.07.2023)

I tried to do it in config, but to no avail.
Do not pay attention about polish locale ;)

Thanks for help :)

**Comment by ankurk91** [comment]
You must report it here
https://github.com/flatpickr/flatpickr
